assets/pages/paper-subscription-landing/paperSubscriptionLandingPageReducer.js

```diff
@@ -2,22 +2,56 @@
 
 // ----- Imports ----- //
 
+import { combineReducers } from 'redux';
 import type { CommonState } from 'helpers/page/commonReducer';
 import { type PaperBillingPlan } from 'helpers/subscriptions';
 import { getQueryParameter } from 'helpers/url';
+import { type PaperDeliveryMethod } from 'helpers/subscriptions';
+
 import { ProductPagePlanFormReducerFor, type State as FormState } from 'components/productPage/productPagePlanForm/productPagePlanFormReducer';
 
+import { type TabActions } from './paperSubscriptionLandingPageActions';
+
+// ----- Tabs ----- //
+
+type ActiveTabState = PaperDeliveryMethod;
+
+const getTabsReducer = (initialTab: PaperDeliveryMethod) =>
+  (state: ActiveTabState = initialTab, action: TabActions): ActiveTabState => {
+
+    switch (action.type) {
+
+      case 'SET_TAB':
+        return action.tab;
+
+      default:
+        return state;
+
+    }
+
+  };
+
+// ----- Exports ----- //
+
 export type State = {
   common: CommonState,
-  page: FormState<PaperBillingPlan>,
+  page: {
+    tab: ActiveTabState,
+    plan: FormState<PaperBillingPlan>,
+  }
 };
 
-
-export default () => {
+export default (initialTab: PaperDeliveryMethod) => {
   const promoInUrl = getQueryParameter('promo');
```

Comment:
Can this be passed as an argument as is the case for `method/initialTab`?

Reply:
yes but that would mean using `getQueryParameter` as as side effect in `paperSubscriptionLandingPage`